sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/routing/MaxStringPartitionKeyComponent.java

```diff
@@ -57,4 +57,15 @@ public IPartitionKeyComponent truncate() {
         return this;
     }
 
+    @Override
+    public Object toObject() {
+        return MaxString.Value;
+    }
+
+    private static class MaxString
```

Comment:
nit: Indentation of curly braces in relevant `*PartitionKeyComponent` classes.

Reply:
fixed